scripts/all.sh

```diff
@@ -2,10 +2,6 @@
 
 set -eu
 
-HERE="$(dirname "$(readlink -f "$BASH_SOURCE")")"
```

Comment:
Why this deletion?

Reply:
Isn't it pointless? If I understand correctly, it reads where the bash is running and does `cd` to that directory. However, we already in that directory. I might be completely wrong, though.

Reply:
It reads where the bash script is defined, not where it is running.